From 7b201cfe92df74f193b8e7bb68c0162ea5fa276e Mon Sep 17 00:00:00 2001
From: nazunalika <tucklesepk@gmail.com>
Date: Fri, 8 Jul 2022 10:27:50 -0700
Subject: [PATCH] Turn on 32 bit builds for armhfp

---
 src/common/engine/i_interface.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/common/engine/i_interface.cpp b/src/common/engine/i_interface.cpp
index 13d35a3..217f921 100644
--- a/src/common/engine/i_interface.cpp
+++ b/src/common/engine/i_interface.cpp
@@ -1,8 +1,6 @@
 #include "i_interface.h"
 #include "st_start.h"
 
-static_assert(sizeof(void*) == 8, "32 builds are not supported");
-
 // Some global engine variables taken out of the backend code.
 FStartupScreen* StartWindow;
 SystemCallbacks sysCallbacks;
-- 
2.31.1

