--- gzdoom/asmjit/CMakeLists.txt	2019-02-25 15:40:17.195766808 -0700
+++ gzdoomnew/asmjit/CMakeLists.txt	2019-02-25 17:58:00.964376439 -0700
@@ -97,7 +97,7 @@
     asmjit/x86/x86regalloc.cpp
 )
 
-add_library(${ASMJITNAME} ${ASMJIT_SRCS} ${ASMJIT_PUBLIC_HDRS})
+add_library(${ASMJITNAME} STATIC ${ASMJIT_SRCS} ${ASMJIT_PUBLIC_HDRS})
 
 set_target_properties(${ASMJITNAME} PROPERTIES OUTPUT_NAME asmjit)
 
