--- gzdoom-g4.0.0-orig/asmjit/CMakeLists.txt	2019-04-07 00:11:51.000000000 -0700
+++ gzdoom-g4.0.0/asmjit/CMakeLists.txt	2019-04-16 11:57:58.349522685 -0700
@@ -97,7 +97,7 @@
     asmjit/x86/x86regalloc.cpp
 )
 
-add_library(${ASMJITNAME} ${ASMJIT_SRCS} ${ASMJIT_PUBLIC_HDRS})
+add_library(${ASMJITNAME} STATIC ${ASMJIT_SRCS} ${ASMJIT_PUBLIC_HDRS})
 
 set_target_properties(${ASMJITNAME} PROPERTIES OUTPUT_NAME asmjit)
 
--- gzdoom-g4.0.0-orig/dumb/CMakeLists.txt	2019-04-07 00:11:51.000000000 -0700
+++ gzdoom-g4.0.0/dumb/CMakeLists.txt	2019-04-16 11:50:31.315715842 -0700
@@ -24,7 +24,7 @@
 
 include_directories( include )
 
-add_library( dumb
+add_library( dumb STATIC
     src/core/unload.c
     src/core/rendsig.c
     src/core/rendduh.c
--- gzdoom-g4.0.0-orig/gdtoa/CMakeLists.txt	2019-04-07 00:11:51.000000000 -0700
+++ gzdoom-g4.0.0/gdtoa/CMakeLists.txt	2019-04-16 11:51:11.654326836 -0700
@@ -35,7 +35,7 @@
 	set( GEN_FP_DEPS ${CMAKE_CURRENT_BINARY_DIR}/arith.h ${CMAKE_CURRENT_BINARY_DIR}/gd_qnan.h )
 endif()
 
-add_library( gdtoa
+add_library( gdtoa STATIC
 	${GEN_FP_FILES}
 	dmisc.c
 	dtoa.c
--- gzdoom-g4.0.0-orig/lzma/CMakeLists.txt	2019-04-07 00:11:51.000000000 -0700
+++ gzdoom-g4.0.0/lzma/CMakeLists.txt	2019-04-16 11:51:36.964710203 -0700
@@ -34,5 +34,5 @@
 	set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_7ZIP_ST" )
 endif()
 
-add_library( lzma ${LZMA_FILES} )
+add_library( lzma STATIC ${LZMA_FILES} )
 target_link_libraries( lzma )
--- gzdoom-g4.0.0-orig/glslang/spirv/CMakeLists.txt	2019-04-07 00:11:51.000000000 -0700
+++ gzdoom-g4.0.0/glslang/spirv/CMakeLists.txt	2019-04-28 11:24:31.063963399 -0700
@@ -44,6 +44,8 @@
     SPVRemapper.h
     doc.h)
 
+set(LIB_TYPE STATIC)
+
 if(ENABLE_AMD_EXTENSIONS)
     list(APPEND
          HEADERS
--- gzdoom-g4.0.0-orig/glslang/glslang/CMakeLists.txt	2019-04-07 00:11:51.000000000 -0700
+++ gzdoom-g4.0.0/glslang/glslang/CMakeLists.txt	2019-04-28 11:27:38.992442803 -0700
@@ -97,6 +97,8 @@
     MachineIndependent/preprocessor/PpContext.h
     MachineIndependent/preprocessor/PpTokens.h)
 
+set(LIB_TYPE STATIC)
+
 # This might be useful for making grammar changes:
 #
 # find_package(BISON)
