--- gzdoom-g4.4.1-orig/libraries/asmjit/CMakeLists.txt	2020-06-10 06:24:21.000000000 -0700
+++ gzdoom-g4.4.1/libraries/asmjit/CMakeLists.txt	2020-06-16 08:54:26.853885875 -0700
@@ -100,6 +100,7 @@
 add_library(${ASMJITNAME} STATIC ${ASMJIT_SRCS} ${ASMJIT_PUBLIC_HDRS})
 
 set_target_properties(${ASMJITNAME} PROPERTIES OUTPUT_NAME asmjit)
+target_link_libraries(${ASMJITNAME} rt)
 
 if(NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL )
     install(TARGETS ${ASMJITNAME}
