--- gzdoom-g4.0.0-orig/asmjit/CMakeLists.txt	2019-04-07 00:11:51.000000000 -0700
+++ gzdoom-g4.0.0/asmjit/CMakeLists.txt	2019-04-16 11:57:58.349522685 -0700
@@ -97,7 +97,7 @@
     asmjit/x86/x86regalloc.cpp
 )
 
-add_library(${ASMJITNAME} ${ASMJIT_SRCS} ${ASMJIT_PUBLIC_HDRS})
+add_library(${ASMJITNAME} STATIC ${ASMJIT_SRCS} ${ASMJIT_PUBLIC_HDRS})
 
 set_target_properties(${ASMJITNAME} PROPERTIES OUTPUT_NAME asmjit)
 
